mne/tests/test_source_estimate.py

```diff
@@ -122,57 +117,14 @@ def test_volume_stc():
     pytest.raises(ValueError, stc.save, fname_vol, ftype='whatever')
     for ftype in ['w', 'h5']:
         for _ in range(2):
-            fname_temp = op.join(tempdir, 'temp-vol.%s' % ftype)
+            fname_temp = os.path.join(tempdir, 'temp-vol.%s' % ftype)
             stc_new.save(fname_temp, ftype=ftype)
             stc_new = read_source_estimate(fname_temp)
             assert (isinstance(stc_new, VolSourceEstimate))
             assert_array_equal(stc.vertices, stc_new.vertices)
             assert_array_almost_equal(stc.data, stc_new.data)
 
 
-@testing.requires_testing_data
-@requires_nibabel()
-def test_save_vol_stc_as_nifti():
-    """Save the stc as a nifti file and export."""
-    import nibabel as nib
-    tempdir = _TempDir()
-    with warnings.catch_warnings(record=True):
-        warnings.simplefilter('always')
-        src = read_source_spaces(fname_vsrc)
-    vol_fname = op.join(tempdir, 'stc.nii.gz')
-
-    # now let's actually read a MNE-C processed file
-    stc = read_source_estimate(fname_vol, 'sample')
-    assert (isinstance(stc, VolSourceEstimate))
-
-    stc.save_as_volume(vol_fname, src,
-                       dest='surf', mri_resolution=False)
-    with warnings.catch_warnings(record=True):  # nib<->numpy
-        img = nib.load(vol_fname)
-    assert (img.shape == src[0]['shape'] + (len(stc.times),))
-
-    with warnings.catch_warnings(record=True):  # nib<->numpy
-        t1_img = nib.load(fname_t1)
-    stc.save_as_volume(op.join(tempdir, 'stc.nii.gz'), src,
-                       dest='mri', mri_resolution=True)
-    with warnings.catch_warnings(record=True):  # nib<->numpy
-        img = nib.load(vol_fname)
-    assert (img.shape == t1_img.shape + (len(stc.times),))
-    assert_allclose(img.affine, t1_img.affine, atol=1e-5)
-
-    # export without saving
-    img = stc.as_volume(src, dest='mri', mri_resolution=True)
-    assert (img.shape == t1_img.shape + (len(stc.times),))
-    assert_allclose(img.affine, t1_img.affine, atol=1e-5)
-
-    src = SourceSpaces([src[0], src[0]])
-    stc = VolSourceEstimate(np.r_[stc.data, stc.data],
-                            [stc.vertices, stc.vertices],
-                            tmin=stc.tmin, tstep=stc.tstep)
-    img = stc.as_volume(src, dest='mri', mri_resolution=False)
-    assert (img.shape == src[0]['shape'] + (len(stc.times),))
```

Comment:
@TommyClausner did you move this test?

if you modify the existing tests it makes it hard to know if you broke something.
I therefore increases the risk unnecessarily.

Reply:
but but but wait... I mean, yes I moved the tests, but don't you think this makes sense? I mean I didn't modify them except for replacing assert_equal with assert

Reply:
yes it makes sense except it makes reviewing more difficult. So we can do this in 2 steps if it helps reviewing process.

Reply:
honestly I think it would be safer to put them back to where they were. You can use the -k option of pytest to decide to run a specific test. I want to avoid us overlooking an API breakage.

thanks